docs/application/dotnet/guides/uix/tts.md

```diff
@@ -300,6 +328,23 @@ To obtain the current state, the supported voice list, and the current voice:
     }
     ```
 
+-   Retrieve the current TTS service state by using the `CurrentServiceState` property of the [Tizen.Uix.Tts.TtsClient](/application/dotnet/api/TizenFX/latest/api/Tizen.Uix.Tts.TtsClient.html) class:
+
```

Comment:
This looks good.
Additionally, how about adding more descriptions of the current service state in ## TTS information retrieval??
Also, please change `TTS daemon` to `TTS service`, because TTS service engine is not a daemon any longer.

Reply:
Thank you for your opinion.
I agree you. TTS daemon no longer exist and service application is used for TTS framework.
I will fix it and I will add the information about the service state into information retrieval.